scripts/acceptance_tool_use.py

```
@@ -0,0 +1,471 @@
+from tests.external_mcp_test_support import manifest
+from tests.integration.test_external_mcp_real_servers_integration import (
+    everything_http,
+    free_port,
+    stop_process,
+)
+
+
+def public_evidence(text: str) -> str:
+    """公開用の写しだけからローカルpathと動的な接続先を除く。"""
+    text = text.replace(str(ROOT), ".")
+    text = re.sub(r"/tmp/ds-tool-182-[^/\s\"']+", "<test-root>", text)
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/ds-tool-182-[^/\\s\\\"\']+"

(S108)